client/signup/steps/site-information/index.jsx

```diff
@@ -112,11 +124,22 @@ export class SiteInformation extends Component {
 		}
 	}
 
-	renderSubmitButton = () => (
-		<Button primary type="submit" onClick={ this.handleSubmit }>
-			{ this.props.translate( 'Continue' ) }
-		</Button>
-	);
+	renderSubmitButton = () => {
+		const { translate } = this.props;
+
+		return (
+			<>
+				<Button primary type="submit" onClick={ this.handleSubmit }>
+					{ translate( 'Continue' ) }
+				</Button>
+				{ abtest( 'skipBusinessInformation' ) === 'show' && (
+					<Button className="site-information__skip-button" borderless onClick={ this.handleSkip }>
+						{ translate( 'Skip' ) }
```

Comment:
It's a minor thing. Can we provide context for this simple string, so translators can recognize what the button means?